lua/tardis/interiors/modules/sv_spawn_tardis.lua

```diff
@@ -49,6 +49,7 @@ function TARDIS:SpawnTARDIS(ply, customData)
     end
     undo.SetCustomUndoText("Undone " .. printName)
     undo.Finish(printName)
+    entity:EmitSound(entity.metadata.Exterior.Sounds.Spawn)
```

Comment:
This should be a clientside setting and thus the sound must be generated clientside, on an Initialize hook would make sense. It also needs to adhere to the global sound setting as well